src/lib/payments/publish-update.ts

```
@@ -0,0 +1,15 @@
+import type { PaymentUpdate, PaymentUpdateChannel } from '../types'
+
+import { redisPublisher } from '../upstash-redis'
+
+export async function publishPaymentUpdate(paymentId: string, update: PaymentUpdate) {
+  const channel: PaymentUpdateChannel = `paymentId:${paymentId}`
+  const message = JSON.stringify(update)
+  console.log(`Publishing update for payment: "${channel} "\nmessage: ${message}`)
+  try {
+    await redisPublisher.publish(channel, message)
+  } catch (error) {
```

ESLint

[error] Closing curly brace appears on the same line as the subsequent block.

(style/brace-style)